packages/components/data-table/src/data-table.tsx

```diff
@@ -156,21 +157,21 @@ export type TColumn = {
   shouldIgnoreRowClick?: boolean;
 };
 
-export type TDataTable = {
+export type TDataTable<RowData> = {
```

Comment:
See my other comment above

```suggestion
export type TDataTableProps<Row extends TRow = TRow> = {
```